.travis.yml

```diff
@@ -5,12 +5,13 @@ matrix:
     include:
         - php: 5.3
           env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
-        - php: 7.0
         - php: hhvm
         - php: 5.4
         - php: 5.5
         - php: 5.6
         - php: 7.0
+          env: LINT_TEST_CASES=1
+        - php: 7.0
```

Comment:
I would start with the special ones (lowest deps, highest deps with linting, hhvm).
Then the regular ones (5.4, 5.5, 5.6).
Then the additional check for php 7 (with sf 2.8).